hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestOmDBInsightEndPoint.java

```diff
@@ -693,4 +694,59 @@ public void testGetDeletedDirInfo() throws Exception {
     Assertions.assertEquals("/sampleVol/bucketOne/dir_two",
         keyInsightInfoResp.getLastKey());
   }
+
+  @Test
+  public void testGetDirectorySizeInfo() throws Exception {
```

Comment:
This test is based on assumption that NSSummary gets updated when a directory gets deleted and its the sum of size of all files inside a given directory. Can you add an integration test along with Mini cluster  in "TestReconWithOzoneManagerFSO" because test is missing in this integration test for validating the same. Integration Test should include deleting a directory with some set of files and after deletion, existing namespace summary should still provide the size of deleted directory as equivalent of size of all files inside that respective dorectory.

Reply:
Done!